msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -36,6 +36,7 @@ The granular, Teams-specific, RSC permissions define what an application can do
 |TeamsTab.ReadWrite.Group|Update this team's tabs. |
 |TeamsTab.Delete.Group|Delete this team's tabs. |
 |TeamMember.Read.Group|Get this team's members. |
+|TeamsActivity.Send.Group|Send activity notification to team's members. |
```

Comment:
For this page, please use the text from the column titled Teams Client App Details Dialog Text https://microsoft.sharepoint.com/:x:/t/MicrosoftGraph18/Ee1frP1eGWlBnOwMZcxHi9UBBTVupvpx6t7qH9hD-e1fug?e=gX9bri